packages/admin/src/pages/scheme/[schemeId]/editors.getServerSideProps.tsx

```diff
@@ -68,6 +68,7 @@ const getEditorsServerSideProps = async ({
   const { schemeId } = params as Record<string, string>;
   const sessionCookie = getSessionIdFromCookies(req);
   const userServiceJwt = getUserTokenFromCookies(req);
+  const queryParams = new URLSearchParams(req.url?.split('?')[1] ?? '');
```

Comment:
Can we use a safer method of getting the query string? eg https://nextjs.org/docs/app/api-reference/functions/use-router